src/components/contextual/pages/pool/risks/pool-risks.ts

```diff
@@ -67,6 +75,24 @@ export function riskLinks(pool: Pool) {
   return result;
 }
 
+export function generateThirdPartyComposabilityRisks(pool): Risk | undefined {
+  const protocols = boostedProtocols(pool);
+  if (
```

Comment:
@pkattera @garethfuller Here we are explicitly adding Aaeve when the pool has Reaper in its metadata (as Reaper harvests Aave rewards). Do we have more cases like this one?

Reply:
I'm not sure. I believe you've had further discussion on this in slack.

Reply:
Yeah, I forgot to update this conversation. The rules are documented in the unit tests.